nca/FWRules/InteractiveConnectivityGraph.py

```diff
@@ -59,13 +61,13 @@ class ElementRelations:
         highlights: set = field(default_factory=set)
         explanation: list = field(default_factory=set)
 
-    def __init__(self, svg_file_name, output_directory):
+    def __init__(self, svg_file_name, output_directory, expl_xml=None):
```

Comment:
update `init` params in the documentation

Reply:
done